src/lib/KafkaFactory.ts

```diff
@@ -39,4 +41,4 @@ class KafkaFactory {
   }
 }
 
-module.exports = KafkaFactory;
+export default KafkaFactory;
```

Comment:
I've also moved the export notation to where the class is defined in: https://github.com/nodefluent/kafka-streams/pull/166